clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -10366,7 +10378,9 @@ void OffloadPackager::ConstructJob(Compilation &C, const JobAction &JA,
           AL += " ";
           AL += A;
         }
-        Parts.emplace_back(C.getArgs().MakeArgString(Twine(Opt) + AL));
+        for (StringRef Split : llvm::split(AL, ',')) {
```

Comment:
Could you please add a comment describing what this newly added code does or update the above existing comment.

Reply:
Added a comment